From 16a57c9f8b541d467e9590258e35d535d3f1ff45 Mon Sep 17 00:00:00 2001
From: Corey Bryant <corey.bryant@canonical.com>
Date: Fri, 27 Jan 2023 17:02:35 -0500
Subject: [PATCH 06/86] Revert "Bug#34834787 Typo in error message when
 bootstrap servers list is empty"

This reverts commit 3028dac85189758ac95af6a26d018ee0cb104bb4.
---
 router/src/metadata_cache/src/metadata_cache_plugin.cc     | 2 +-
 router/tests/component/test_router_configuration_errors.cc | 4 ++--
 router/tests/component/test_state_file.cc                  | 2 +-
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/router/src/metadata_cache/src/metadata_cache_plugin.cc b/router/src/metadata_cache/src/metadata_cache_plugin.cc
index 22c64a3c13e..b0a8a1ca7a0 100644
--- a/router/src/metadata_cache/src/metadata_cache_plugin.cc
+++ b/router/src/metadata_cache/src/metadata_cache_plugin.cc
@@ -200,7 +200,7 @@ static void start(mysql_harness::PluginFuncEnv *env) {
         (!config.metadata_cache_dynamic_state ||
          config.metadata_cache_dynamic_state->get_metadata_servers().empty())) {
       throw std::runtime_error(
-          "list of metadata-servers is empty: 'bootstrap_server_addresses' in the configuration file is empty or not set and "s +
+          "list of metadata-servers is empty: 'bootstrap_server_addresses' is the configuration file is empty or not set and "s +
           (!config.metadata_cache_dynamic_state
                ? "no known 'dynamic_config'-file"
                : "list of 'cluster-metadata-servers' in 'dynamic_config'-file "
diff --git a/router/tests/component/test_router_configuration_errors.cc b/router/tests/component/test_router_configuration_errors.cc
index 65e7d852873..bfa3e58b590 100644
--- a/router/tests/component/test_router_configuration_errors.cc
+++ b/router/tests/component/test_router_configuration_errors.cc
@@ -234,7 +234,7 @@ static const BrokenConfigParams broken_config_params[]{
                                                          {"user", "foobar"},
                                                      }),
      },
-     "list of metadata-servers is empty: 'bootstrap_server_addresses' in the "
+     "list of metadata-servers is empty: 'bootstrap_server_addresses' is the "
      "configuration file is empty or not set and no known "
      "'dynamic_config'-file",
      ""},
@@ -247,7 +247,7 @@ static const BrokenConfigParams broken_config_params[]{
                  {"bootstrap_server_addresses", ""},
              }),
      },
-     "list of metadata-servers is empty: 'bootstrap_server_addresses' in the "
+     "list of metadata-servers is empty: 'bootstrap_server_addresses' is the "
      "configuration file is empty or not set and no known "
      "'dynamic_config'-file",
      ""},
diff --git a/router/tests/component/test_state_file.cc b/router/tests/component/test_state_file.cc
index 73ef4f275ab..2d70c6e7247 100644
--- a/router/tests/component/test_state_file.cc
+++ b/router/tests/component/test_state_file.cc
@@ -700,7 +700,7 @@ TEST_F(StateFileDynamicChangesTest, EmptyMetadataServersList) {
   // proper error should get logged
   EXPECT_TRUE(wait_log_file_contains(
       router,
-      "'bootstrap_server_addresses' in the configuration file is empty "
+      "'bootstrap_server_addresses' is the configuration file is empty "
       "or not set and list of 'cluster-metadata-servers' in "
       "'dynamic_config'-file is empty, too.",
       3 * kTTL));
-- 
2.37.2

